src/build123d/exporters.py

```diff
@@ -1187,6 +1187,9 @@ def _line_element(self, edge: Edge) -> ET.Element:
 
     def _circle_segments(self, edge: Edge, reverse: bool) -> list[PathSegment]:
         # pylint: disable=too-many-locals
+        if edge.length < 1e-6:
+            print("Skipping arc that is too small to export safely (length < 1e-6)")
```

Comment:
Shouldn't this be a warning? Something like:
```py
            warnings.warn("Skipping arc that is too small to export safely (length < 1e-6)", stacklevel=2)
```
I'm not sure what the `stacklevel` should be here - I guess it should point at the `export_svg` call.

Reply:
I had it as a warning before but the warning came up when I ran the test suit which I didn't think was the desired outcome. Do I need to change the test somehow so that the warning is not shown when the whole test suit is run?

Reply:
The test can be written like this:
```py
        with self.assertWarns(UserWarning):
            Wire.make_rect(1, 1).vertex()
```